client/src/components/EditRecipe.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import RecipesForm from './RecipesForm';
+import { startEditRecipe } from '../actions/recipes';
```

Comment:
What's the purpose of the name `startEditRecipe`? Is it that editing a recipe is a process that doesn't end here?